### PR TITLE
Improve phpstan config

### PR DESCRIPTION
**Pull request by szepeviktor**

### CODE REVIEW

File: phpstan.neon [hunk 1]
```diff
@@ -1,12 +1,10 @@
...
   paths:
-  - src
-  - tests
+    - %rootDir%/../../../src/
+    - %rootDir%/../../../tests/
   excludes_analyse:
-  - %rootDir%/../../../src/example/*
-  - %rootDir%/../../../src/resources/views/*
-  - %rootDir%/../../../tests/Support/database/*
+    - %rootDir%/../../../tests/Support/database/
```
Comment (mfn) [inline phpstan.neon]: I believe I should switch to `%currentWorkingDir%`; `%rootDir%` doesn't work with Larastan without adjusting which seems odd. currentWorkingDir seems more "stable" as the analyser is always meant to be run from the projects root anyway

Comment (szepeviktor) [reply phpstan.neon]: Okay!

**Comment by mfn** [comment]
Added the variable change and force-pushed to be in sync with updated master

**Comment by mfn** [comment]
Thanks @szepeviktor !

**Comment by szepeviktor** [comment]
You're welcome.

**Pull request merged**